src/darwin/Framework/CHIP/MTRCluster.h

```diff
@@ -144,6 +154,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, copy) NSNumber * maxInterval;
 
+/**
+ * Sets IsUrgent flag of EventPathIB in the Subscribe Request. The default value is NO.
```

Comment:
Default needs to be YES to match current behavior of `subscribeWithQueue`.

Also, we should be using this value in `subscribeWithQeuue`, now that we have it, and remove the hardcoded `true` there.